README.md

```diff
@@ -1,2 +1,53 @@
 # Weather-App-JS
+
 Creating a weather forecast site using the weather API . The user is able to search for a specific location to know the temperature and toggle displaying the data in Fahrenheit or Celsius. Also, this App shows other weather parameters.
+
+![Weather App](./src/Screenshot.png?raw=true "Main Page")
+
+## Built With
+ - HTML
+ - CSS
+ - Bootstrap
+ - Javascript
+ - Webpack
+
+
+## Live Demo
+
+[Live demo](https://laguna1.github.io/Weather-App-JS/)
```

Comment:
## Fix link to live version
- [ ] The link is meant to take one to the live version of the app (not the readme). Kindly fix this.